<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>458</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>7</hsizetype>
    <vsizetype>7</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="windowIcon" >
   <iconset>../images/pootling.png</iconset>
  </property>
  <property name="statusTip" >
   <string/>
  </property>
  <property name="whatsThis" >
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>597</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuOpen_Recent" >
     <property name="title" >
      <string>Open &amp;Recent</string>
     </property>
     <property name="icon" >
      <iconset>../images/open.png</iconset>
     </property>
     <addaction name="actionClear" />
     <addaction name="separator" />
    </widget>
    <addaction name="actionOpenInNewWindow" />
    <addaction name="actionOpen" />
    <addaction name="menuOpen_Recent" />
    <addaction name="action_Close" />
    <addaction name="separator" />
    <addaction name="actionSave" />
    <addaction name="actionSaveas" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
    <addaction name="separator" />
    <addaction name="actionCut" />
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
    <addaction name="separator" />
    <addaction name="actionSelectAll" />
    <addaction name="separator" />
    <addaction name="actionFind" />
    <addaction name="actionFindPrevious" />
    <addaction name="actionFindNext" />
    <addaction name="actionReplace" />
    <addaction name="separator" />
    <addaction name="actionCopySource2Target" />
    <addaction name="actionToggleFuzzy" />
    <addaction name="actionEdit_Header" />
   </widget>
   <widget class="QMenu" name="menuView" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="title" >
     <string>&amp;View</string>
    </property>
    <addaction name="actionFilterFuzzy" />
    <addaction name="actionFilterTranslated" />
    <addaction name="actionFilterUntranslated" />
   </widget>
   <widget class="QMenu" name="menuGo" >
    <property name="title" >
     <string>&amp;Go</string>
    </property>
    <addaction name="actionFirst" />
    <addaction name="actionPrevious" />
    <addaction name="actionNext" />
    <addaction name="actionLast" />
    <addaction name="separator" />
    <addaction name="actionGoTo" />
   </widget>
   <widget class="QMenu" name="menuBookmark" >
    <property name="title" >
     <string>&amp;Bookmarks</string>
    </property>
    <addaction name="actionAddBookmarks" />
    <addaction name="actionClearBookmarks" />
    <addaction name="separator" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutQT" />
   </widget>
   <widget class="QMenu" name="menuSettings" >
    <property name="title" >
     <string>&amp;Settings</string>
    </property>
    <addaction name="actionPreferences" />
    <addaction name="action_TM" />
   </widget>
   <widget class="QMenu" name="menu_Tool" >
    <property name="title" >
     <string>&amp;Tool</string>
    </property>
    <widget class="QMenu" name="menu_TM" >
     <property name="title" >
      <string>&amp;TM</string>
     </property>
     <property name="icon" >
      <iconset>../images/memory.png</iconset>
     </property>
     <addaction name="action_lookup_Text" />
     <addaction name="actionAuto_translate" />
    </widget>
    <addaction name="separator" />
    <addaction name="menu_TM" />
    <addaction name="actionCatalogManager" />
   </widget>
   <widget class="QMenu" name="menuWindow" >
    <property name="title" >
     <string>&amp;Window</string>
    </property>
    <widget class="QMenu" name="menuToolbars" >
     <property name="title" >
      <string>&amp;Toolbars</string>
     </property>
    </widget>
    <addaction name="separator" />
    <addaction name="separator" />
    <addaction name="menuToolbars" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuView" />
   <addaction name="menuGo" />
   <addaction name="menuBookmark" />
   <addaction name="menu_Tool" />
   <addaction name="menuSettings" />
   <addaction name="menuWindow" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolStandard" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="windowTitle" >
    <string>&amp;Standard Toolbar</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionSave" />
   <addaction name="separator" />
   <addaction name="actionCut" />
   <addaction name="actionCopy" />
   <addaction name="actionPaste" />
   <addaction name="separator" />
   <addaction name="actionUndo" />
   <addaction name="actionRedo" />
  </widget>
  <widget class="QToolBar" name="toolNavigation" >
   <property name="contextMenuPolicy" >
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops" >
    <bool>false</bool>
   </property>
   <property name="windowTitle" >
    <string>&amp;Navigation Toolbar</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionFirst" />
   <addaction name="actionPrevious" />
   <addaction name="actionNext" />
   <addaction name="actionLast" />
  </widget>
  <widget class="QToolBar" name="toolFilter" >
   <property name="windowTitle" >
    <string>&amp;Filter Toolbar</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
   <addaction name="actionFilterFuzzy" />
   <addaction name="actionFilterTranslated" />
   <addaction name="actionFilterUntranslated" />
  </widget>
  <action name="actionNew" >
   <property name="icon" >
    <iconset>../images/new.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;New</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset>../images/open.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>open a file&lt;/h3>You will be asked for the name of a file to be opened and open recent file in an editor window.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/save.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Save a file&lt;/h3>Save the contents of current editor window.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveas" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/saveAs.png</iconset>
   </property>
   <property name="text" >
    <string>Save &amp;As</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset>../images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip" >
    <string>Quit the application</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Quit the application&lt;/h3>You will be asked to save the opened file that is being edited.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/undo.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Undo</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Undo&lt;/h3>Undo the last change done in the current editor.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/redo.png</iconset>
   </property>
   <property name="text" >
    <string>Re&amp;do</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Redo&lt;/h3>Redo the last change done in the current editor.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/cut.png</iconset>
   </property>
   <property name="text" >
    <string>Cu&amp;t</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Cut&lt;/h3>Cut the selected text of the current editor to the clipboard.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/copy.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Copy</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Copy&lt;/h3>Copy the selected text of the current editor to the clipboard.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/paste.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Paste</string>
   </property>
   <property name="iconText" >
    <string>Paste</string>
   </property>
   <property name="toolTip" >
    <string>Paste</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Paste&lt;/h3>Paste the last cut/copies text from the clipboard to the current editor.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionFind" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/find.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Find</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="actionShowMenuBar" >
   <property name="text" >
    <string>Show MenuBar</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionShowTUview" >
   <property name="text" >
    <string>Show TUview</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAboutQT" >
   <property name="text" >
    <string>About &amp;QT</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="actionShow_MenuBar" >
   <property name="text" >
    <string>Show MenuBar</string>
   </property>
  </action>
  <action name="actionShow_TUview" >
   <property name="text" >
    <string>Show TUview</string>
   </property>
  </action>
  <action name="actionFirst" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/first.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;First</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>First&lt;/h3>Move to first row of the table.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+PgUp</string>
   </property>
  </action>
  <action name="actionPrevious" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/back.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Previous</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Previous&lt;/h3>Move to previous row inside the table.</string>
   </property>
   <property name="shortcut" >
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionNext" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/forward.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Next</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Next&lt;/h3>Move to next row inside the table.</string>
   </property>
   <property name="shortcut" >
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionLast" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/last.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Last</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Last&lt;/h3>Move to last row of the table.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+PgDown</string>
   </property>
  </action>
  <action name="actionCopySource2Target" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Cop&amp;y Source to Target</string>
   </property>
   <property name="shortcut" >
    <string>F2</string>
   </property>
  </action>
  <action name="actionToggleFuzzy" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>To&amp;ggle fuzzy</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionFile" >
   <property name="text" >
    <string>file</string>
   </property>
  </action>
  <action name="actionFind_Previous" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Find Previous</string>
   </property>
  </action>
  <action name="actionFind_Next" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Find Next</string>
   </property>
  </action>
  <action name="actionReplace" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/replace.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Replace</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFindNext" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/next.png</iconset>
   </property>
   <property name="text" >
    <string>Find &amp;Next</string>
   </property>
   <property name="iconText" >
    <string>Find Next</string>
   </property>
   <property name="toolTip" >
    <string>Find Next</string>
   </property>
   <property name="shortcut" >
    <string>F3</string>
   </property>
  </action>
  <action name="actionOpenInNewWindow" >
   <property name="icon" >
    <iconset>../images/window_new.png</iconset>
   </property>
   <property name="text" >
    <string>Open in New Window</string>
   </property>
   <property name="visible" >
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFindPrevious" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/previous.png</iconset>
   </property>
   <property name="text" >
    <string>Find Pre&amp;vious</string>
   </property>
   <property name="shortcut" >
    <string>Shift+F3</string>
   </property>
  </action>
  <action name="actionSelectAll" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Select &amp;All</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionEdit_Header" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;Header...</string>
   </property>
   <property name="statusTip" >
    <string>Open the dialog to edit the header information.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionPreferences" >
   <property name="icon" >
    <iconset>../images/configure.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Preferences...</string>
   </property>
  </action>
  <action name="actionToolbars" >
   <property name="text" >
    <string>Toolbars</string>
   </property>
  </action>
  <action name="actionFilterFuzzy" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/fuzzy.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Fuzzy</string>
   </property>
   <property name="toolTip" >
    <string>Hide/Show Fuzzy Items</string>
   </property>
   <property name="statusTip" >
    <string>Hide/Show Fuzzy Items</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Alt+F</string>
   </property>
  </action>
  <action name="actionFilterTranslated" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/translated.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Translated</string>
   </property>
   <property name="toolTip" >
    <string>Hide/Show Translated Items</string>
   </property>
   <property name="statusTip" >
    <string>Hide/Show Translated Items</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Alt+T</string>
   </property>
  </action>
  <action name="actionFilterUntranslated" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/untranslated.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Untranslated</string>
   </property>
   <property name="toolTip" >
    <string>Hide/Show Untranslated Items</string>
   </property>
   <property name="statusTip" >
    <string>Hide/Show Untranslated Items</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Alt+U</string>
   </property>
  </action>
  <action name="action_Close" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/fileclose.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Close</string>
   </property>
   <property name="statusTip" >
    <string>Close the current opened file</string>
   </property>
   <property name="whatsThis" >
    <string>&lt;h3>Close the current opened file&lt;/h3>You will be asked whether to save the current opened file.</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClear" >
   <property name="text" >
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="actionGoTo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>&amp;GoTo Line</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionAddBookmarks" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset>../images/bookmarkadd.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Add Bookmarks</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionClearBookmarks" >
   <property name="text" >
    <string>&amp;Clear Bookmarks</string>
   </property>
   <property name="shortcut" >
    <string/>
   </property>
  </action>
  <action name="action_lookup_Text" >
   <property name="text" >
    <string>&amp;Lookup Unit</string>
   </property>
  </action>
  <action name="actionAuto_translate" >
   <property name="text" >
    <string>&amp;Auto Translate</string>
   </property>
  </action>
  <action name="action_TM" >
   <property name="icon" >
    <iconset>../images/memory.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;TM</string>
   </property>
  </action>
  <action name="actionCatalogManager" >
   <property name="text" >
    <string>Catalog Manager</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
